landscape/explanation/glossary/ppa.md

```diff
@@ -0,0 +1,6 @@
+# PPA 
+
+A **Personal Package Archive (PPA)** is a software repository specifically made for Ubuntu users. PPAs are used by developers to distribute their software. When you install Landscape, you provide a PPA so your system knows where to download Landscape and its required packages. For example, if you want the beta version of Landscape, you must add the Landscape beta PPA to your system.
+
+It's important to use the correct PPA when you install Landscape. This is because there are different PPAs for different versions of Landscape 
```

Comment:
```suggestion
It's important to use the correct PPA when you install Landscape. This is because there are different PPAs for different versions of Landscape. For example, the Landscape beta PPA is `ppa:landscape/self-hosted-beta`, and the PPA for Landscape 24.04 LTS is `ppa:landscape/self-hosted-24.04`. Before you install Landscape, make sure you select the desired PPA for your system.
```